python/ml4ir/data/tfrecord_reader.py

```diff
@@ -9,39 +9,56 @@
 from ml4ir.features.feature_config import FeatureConfig
 from ml4ir.config.keys import TFRecordTypeKey, FeatureTypeKey
 
-from typing import Union, Optional
+from typing import Optional
 
 """
 This module contains helper methods for reading and writing
 data in the train.SequenceExample protobuf format
 """
 
 
-def make_parse_fn(feature_config: FeatureConfig, max_num_records: int = 25) -> tf.function:
-    """Create a parse function using the context and sequence features spec"""
+def _get_default_value(dtype, serving_info):
+    if dtype == tf.float32:
+        return serving_info.get("default_value", 0.0)
```

Comment:
we should make a note/TODO to make sure we track whether it matters to fill in with values which are valid inputs (instead of known sentinel values which will hopefully never be input).  It may affect per-feature normalization, too?

Reply:
Right now you can define defaults in the feature_config YAML, as part of the serving_info. If that is absent, I use preset default based on the dtype. 
We do not have any normalization yet other than batch norm, but we might in the future, yes.

Reply:
Is there something else you were referring to with the TODO?